packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -607,6 +610,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
         aria-setsize={ totalItemCount }
         onKeyDown={ this._onSplitContainerItemKeyDown.bind(this, item) }
         onClick={ this._executeItemClick.bind(this, item) }
+        onTouchStart={ this._onTouchStart.bind(this) }
```

Comment:
We shouldn't need to bind this, you can use autobind on the function.

Also, why are we always setting this when we might also be setting onpointerdown? Couldn't this just be set in an else above if we know we can't use pointerdown... or is this always set just for automation?

Reply:
From what I know from merging all the changes, we moved away from using autobind and returned to using this, but I'm not sure of the reason. @joschect do you have any insights of why we moved away from this?

You are correct with why I had to set it here. It's for automation, we can't simulate the onTouchStart event any other way.

Reply:
Ah, good call. I didn't realize fabric moved away from autobind, looking at the changelog it states that you should use arrow functions instead of autobind so you could just do this._onTouchStart and then in the function definition use an arrow function like `private _onTouchStart = (ev: TouchEvent<HTMLElement>) => {`

Reply:
@chang47 we've moved away from autobind because it adds general code bloat and increases the size of functions